### PR TITLE
Update waku nodes

### PR DESCRIPTION
**Pull request by cammellos**
Update waku nodes on login/creation of account

### CODE REVIEW

File: api/defaults.go [hunk 1]
```diff
@@ -26,6 +26,20 @@ const defaultKeycardPairingDataFile = "/ethereum/mainnet_rpc/keycard/pairings.js
 
 var paths = []string{pathWalletRoot, pathEIP1581, pathDefaultChat, pathDefaultWallet}
 
+const (
+	statusProdFleet = "status.prod"
+	statusTestFleet = "status.test"
+	wakuv2ProdFleet = "wakuv2.prod"
+	wakuv2TestFleet = "wakuv2.test"
+)
```
Comment (richard-ramos) [inline api/defaults.go:34]: These fleets are already defined in params/cluster.go

File: api/defaults.go [hunk 2]
```diff
@@ -88,6 +102,19 @@ func defaultSettings(generatedAccountInfo generator.GeneratedAccountInfo, derive
 	return s, nil
 }
 
+func setDefaultFleet(nodeConfig *params.NodeConfig) error {
+	clusterConfig, err := params.LoadClusterConfigFromFleet("status.prod")
```
Comment (richard-ramos) [inline api/defaults.go]: You can use the constant for status.prod here

**Review by richard-ramos — APPROVED** [review]
LGTM, just left some comments regarding the usage of constants

File: api/defaults.go [hunk 3]
```diff
@@ -26,6 +26,20 @@ const defaultKeycardPairingDataFile = "/ethereum/mainnet_rpc/keycard/pairings.js
...
 
+const (
+	statusProdFleet = "status.prod"
+	statusTestFleet = "status.test"
+	wakuv2ProdFleet = "wakuv2.prod"
+	wakuv2TestFleet = "wakuv2.test"
+)
+
+var defaultWakuNodes = map[string][]string{
+	statusProdFleet: []string{"enrtree://AL65EKLJAUXKKPG43HVTML5EFFWEZ7L4LOKTLZCLJASG4DSESQZEC@prod.status.nodes.status.im"},
```
Comment (qfrank) [inline api/defaults.go:37]: just wondering what's the difference between `enrtree://AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM@prod.nodes.status.im` and `enrtree://AL65EKLJAUXKKPG43HVTML5EFFWEZ7L4LOKTLZCLJASG4DSESQZEC@prod.status.nodes.status.im` ? Do we need update `status-mobile` as well for local pairing ?

**Review by qfrank — APPROVED** [review]

File: api/defaults.go (hunk 3 above)
Comment (jakubgs) [reply api/defaults.go:37]: * https://github.com/status-im/status-mobile/pull/17418
* https://github.com/status-im/status-go/pull/4063

**Review by jakubgs — APPROVED** [review]

**Pull request merged**